test/integration/functional_test.go

```diff
@@ -73,7 +73,7 @@ var runCorpProxy = detect.GithubActionRunner() && runtime.GOOS == "linux" && !ar
 // TestFunctional are functionality tests which can safely share a profile in parallel
 func TestFunctional(t *testing.T) {
 
-	profile := UniqueProfileName("functional")
+	profile := UniqueProfileName("func")
```

Comment:
why this change?

Reply:
The directory path length is too long for QEMU